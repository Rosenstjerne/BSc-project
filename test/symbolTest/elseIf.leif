var int x;
x = 10;

if (x > 5) {
    print(6);
} else {
    print(4);
}
