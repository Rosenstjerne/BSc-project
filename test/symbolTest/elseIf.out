function null main() {
	var int x;
	
	x = 10;
	if((x > 5))
	#bein if
		print(6);
		
	#end if
	#begin else
	else
		print(4);
		
	#end else
	
}
